src/js/menu/menu.js

```diff
@@ -54,6 +65,72 @@ class Menu extends Component {
 
     return el;
   }
+
+  /**
+   * Handle key press for menu
+   *
+   * @param {Object} event Event object
+   * @method handleKeyPress
+   */
+  handleKeyPress (event) {
+    if (event.which === 37 || event.which === 40) { // Left and Down Arrows
+      event.preventDefault();
+      this.stepForward();
+    } else if (event.which === 38 || event.which === 39) { // Up and Right Arrows
```

Comment:
Agreed! That was on my list to look at as park of this accessibility work; any thoughts on where it should go? utils/dom.js, perhaps? We need the ESC key from the modal dialog, too.